pymaker/oracles.py

```diff
@@ -46,7 +47,11 @@ def __init__(self, web3: Web3, address: Address):
     def poke(self) -> Transact:
         return Transact(self, self.web3, self.abi, self.address, self._contract, 'poke', [])
 
-    def __repr__(self):
-        return f"OSM('{self.address}')"
+    def peek(self) -> Wad:
+        return Wad(Web3.toInt(self.web3.eth.getStorageAt(self.address.address, 3)[16:]))
```

Comment:
I'd suggest to we externalize logic of extracting price e.g in `_extract_price(storage)`  function and add unit test for it. Then reuse function in both `peek` and `peep`

Reply:
Done.

Reply:
Done.